tests-system/checks-10/tracing

```diff
@@ -0,0 +1 @@
+LRM.Applicable_Components
```

Comment:
Like with PR #35 please use the `rbt-` naming scheme for new tests that test specific requirements.